README.md

```diff
@@ -83,6 +84,14 @@ If the query can be built with parameters (in this case `id`, `account` and `dat
 as: `${DOCTYPE}/${id}/account/${account}/date/${date}`
 ```
 
+# Import order
+```js
+import libs from externals // (ex: lodash)
+
+import libs from internal // (ex: cozy-client)
+
+import libs from local // (absolute or relative path, depending on the project)
+```
```

Comment:
import libs or components from local // (absolute or relative path, depending on the project)
import styles from local // (absolute or relative path, depending on the project)